### PR TITLE
Let the ByteBuddy library version be controlled by the BOM

### PR DESCRIPTION
**Pull request by Sanne**
As suggested by Alexey in https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/common.20dependencies.20NOT.20in.20the.20BOM

cc/ @beikov

### CODE REVIEW

**Review by aloubyansky — APPROVED** [review]

File: bom/application/pom.xml [hunk 1]
```diff
@@ -90,7 +91,7 @@
         <commons-lang3.version>3.12.0</commons-lang3.version>
         <commons-codec.version>1.15</commons-codec.version>
         <classmate.version>1.5.1</classmate.version>
-        <hibernate-orm.version>5.6.5.Final</hibernate-orm.version>
+        <hibernate-orm.version>5.6.5.Final</hibernate-orm.version> <!-- When updating, align bytebuddy.version to Hibernate needs as well -->
```
Comment (gsmet) [inline bom/application/pom.xml:94]: Usually, in this case, we don't respect the alphabetical order and we add the related dependency just below. But I don't really mind :).

Comment (Sanne) [reply bom/application/pom.xml:94]: ok I'll gladly change that as I'd prefer them close as well.  I had (wrongly) assumed that people wouldn't have liked that :)

**Comment by Sanne** [comment]
pushed a revised version

**Comment by gsmet** [comment]
@aloubyansky is it something you wanted to backport?

**Comment by aloubyansky** [comment]
Let's backport it.

**Pull request merged**